editor/inspector/contributions/asset.js

```diff
@@ -0,0 +1,431 @@
+'use strict';
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Recommend to change these codes into js style, these codes come from tsc.

Reply:
ok